console-subscriber/src/lib.rs

```diff
@@ -5,15 +5,19 @@ use serde::Serialize;
 use std::{
     cell::RefCell,
     fmt,
-    net::{IpAddr, Ipv4Addr, SocketAddr},
+    net::{IpAddr, Ipv4Addr},
     sync::{
         atomic::{AtomicUsize, Ordering},
         Arc,
     },
     time::{Duration, Instant},
 };
 use thread_local::ThreadLocal;
-use tokio::sync::{mpsc, oneshot};
+use tokio::{
+    net::UnixListener,
+    sync::{mpsc, oneshot},
+};
+use tokio_stream::wrappers::UnixListenerStream;
```

Comment:
i believe the `UnixListener` imports won't compile on operating systems other than Unix (e.g., on Windows). we probably need to add a `cfg` attribute so that this code builds on Windows.
```suggestion
use tokio::sync::{mpsc, oneshot};
#[cfg(unix)]
use tokio::net::UnixListener;
#[cfg(unix)]
use tokio_stream::wrappers::UnixListenerStream;
```

Reply:
Done.